react/introduction/setting_up_a_react_environment.md

```diff
@@ -154,5 +154,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- [Vite React App with EsLint & Prettier in VSCode (2023)](https://www.youtube.com/watch?v=SMbqi1HPprc) is a helpful video guide on how to set up Prettier inside your Vite-React projects.
```

Comment:
Why has this content been deleted? Maybe related to a different contribution you've made or something? Seems completely unrelated to your link change in the conclusion lesson

Reply:
This was part of another fix that was merged a while ago [Issue #27878](https://github.com/TheOdinProject/curriculum/pull/27878). I fetched from upstream on my local clone but did not merge hence that commit, my bad.